packages/neos-ui/src/Containers/PrimaryToolbar/PublishDropDown/index.js

```diff
@@ -91,6 +94,7 @@ export default class PublishDropDown extends PureComponent {
             neos
         } = this.props;
 
+        const workspaceModuleUri = $get('routes.core.modules.workspaces', neos);
```

Comment:
This Uri lacks an `management` part, at least for me. I didn't know exactly where it comes from. 
But this results in a link to `neos.org/neos/workspaces/show?moduleArguments[workspace]=user-admin`
and this leads to a 404, whereas in the old ui it is: `neos.org/neos/management/workspaces/show?moduleArguments[workspace]=user-admin`

Reply:
duh, sorry about that, fixed

Reply:
No problem, that what reviews are for. :)